core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -152,4 +153,57 @@ public void validatePropertyKeys() {
   public void testGCDeadServerWaitSecond() {
     assertEquals("1h", Property.GC_WAL_DEAD_SERVER_WAIT.getDefaultValue());
   }
+
+  @SuppressWarnings("deprecation")
```

Comment:
Rather than suppress warnings across the entire method, it's better to only suppress the one case in the method that is deprecated. This avoids masking future un-triaged warnings.

Here, the `Property.INSTANCE_DFS_URI` can be assigned to a variable of type Property just before its corresponding `assertTrue` line, and the assignment can be suppressed.